haas/cli.py

```diff
@@ -215,17 +142,27 @@ def setup_http_client():
                            project_domain_id=os_project_domain_id)
         sess = session.Session(auth=auth)
         http_client = KeystoneHTTPClient(sess)
+        # For calls using the client library
+        C = Client(ep, http_client)
         return
     except (ImportError, KeyError):
         pass
     # Finally, fall back to no authentication:
     http_client = requests.Session()
+    C = Client(ep, http_client)
 
 
 class FailedAPICallException(Exception):
     pass
```

Comment:
I misunderstood your comment last time was pointing at the code that initializes the client library. 
Removed the definition and imported same from the client library. Next commit would have the changes.